main.nf

```diff
@@ -18,10 +18,26 @@ nextflow.enable.dsl = 2
 
 WorkflowMain.initialise(workflow, params, log)
 
-include {METAGENOMICS_SR} from './workflows/metagenomics-sr'
+if (params.platform == 'illumina') {
+    include { ILLUMINA } from './workflows/illumina'
+} else if (params.platform == 'nanopore') {
+    include { NANOPORE } from './workflows/nanopore'
+}
+
+
+workflow ARCADIASCIENCE_METAGENOMICS {
+    //
+    // WORKFLOW: Illumina processing and assembly
+    //
+    if (params.platform == 'illumina') {
```

Comment:
There are no defaults for these, so I'd probably throw an error early in the pipeline if this is `null` or more generically you could add an `else` field.